player.py

```diff
@@ -0,0 +1,47 @@
+import pygame.mixer
+
+
+def produce_music(music_file):
```

Comment:
I don't really like this name. `produce` doesn't sound right here. Maybe just `play`?

Reply:
play is already taken